website/docs/packages/overview.md

```diff
@@ -0,0 +1,82 @@
+# Overview
+
+Reusability is an integral part of application development, offering various advantages like speeding up the development process, centralizing business logic, and introducing standardization into applications.
+
+Appsmith enhances reusability by providing a feature set, allowing you to reuse widgets, datasource, queries, JS logic, and more across your applications.
+
+## Reusability Features
+
+The introduction of Blocks, Modules, and Packages aligns with the diverse needs of developers, community members, and businesses, promoting efficiency and collaboration in application development.
+
+
+### Blocks
+
+Blocks are reusable components in Appsmith, consisting of **widgets**, **queries**, **custom libraries**, **databases**, and/or **JavaScript** bundled into a single **JSON** document. This exportable format allows developers to capture and reuse their work across various applications. 
+
+To reuse entities, just click on the page settings and choose the import/export option. You can select the entities you want to export, allowing you to reuse elements available on that page. This feature proves particularly beneficial for developers looking to avoid the tedious task of copying code across different pages and applications.
+
+<ZoomImage
+  src="/img/blocks-overview.png" 
+  alt=""
+  caption=""
+/>
+
+
+See [How to Reuse Application Entities](https://appsmith-docs-git-feat-partial-import-export-get-appsmith.vercel.app/build-apps/how-to-guides/import-export-app-entities)
```

Comment:
This page is named as "How to Reuse Application Entities". However, it only mention the blocks experience of partial import/export. Modules will be also another way to reuse application entities that also avoids code duplication. 
Suggestion: Should this page have a title specific to how to copy application entities or something similar specific to that feature, or include in this page also information about how to reuse with modules?